get-started/index.md

```diff
@@ -157,11 +172,7 @@ To completely remove Cryostat and all objects and recordings created by it:
     - If the Cryostat Operator has already been uninstalled, please reinstall it
       before deleting the Cryostat custom resource.
 3. Uninstall the Cryostat Operator.
-    - If installed using kubectl, run:
-      ```
-      $ kubectl delete -k github.com/cryostatio/cryostat-operator//config/default?ref=v{{ site.data.versions.cryostat.version }}
-      ```
-    - If installed using the operator bundle, run:
-      ```
-      $ operator-sdk cleanup cryostat-operator
-      ```
+    ```
+    $ kubectl get clusterserviceversion -n my-cryostat-operator
+    $ kubectl delete clusterserviceversion <cryostat-operator-csv-name> -n my-cryostat-operator
```

Comment:
I think they might need to delete the Subscription object as well:
https://olm.operatorframework.io/docs/tasks/uninstall-operator/#combine-steps-2-and-3

This still missing some things created during installation though (e.g. Namespace, OperatorGroup):
https://operatorhub.io/how-to-install-an-operator#What-happens-when-I-execute-the-'Install'-command-presented-in-the-pop-up?

To do a full cleanup of the operator including the Namespace, they could do:
`kubectl delete -f https://operatorhub.io/install/cryostat-operator.yaml`